debezium-schema-generator/pom.xml

```diff
@@ -65,6 +84,41 @@
         </plugin>
       </plugins>
     </pluginManagement>
+    <plugins>
+      <plugin>
```

Comment:
Could you provide some background why the fat JAR is needed? I know you mentioned it yesterday, but I forgot :/

Reply:
If we truly need it, can you please add the plug-in versions to the versions / plug-in management blocks in the build parent pom rather than here?

Reply:
When using the plugin downstream I was not able to provide the dependencies on the classpath because of the static configured classpath before. I don't know if it resolves now?

Reply:
Try it out and we'll see :)

Reply:
@gunnarmorling is there no other way than to define all connectors plus `org.apache.kafka.connect-api` as "provided dependencies in a downstream build?

Reply:
@rk3rn3r, so having to define the actual connectors for downstream makes sense to me. It's the lever for controlling which files you want to generate. In regards to `connect-api`, how about you add it as a dependency to the generator plug-in? Then this should be pulled in automatically now.